docs/synapse/glossary.rst

```diff
@@ -411,7 +411,7 @@ Within Synapse, forking refers to the process of creating a new, empty, writable
 the fork's original :ref:`gloss-view`. The writable layer from the original view becomes read-only with
 respect to the fork. Any changes made within a forked view are made within the new writable layer. These
 changes can optionally be merged back into the original view, or discarded. (Note that any view-specific
-automation, such as triggers, dmons, or cron jobs, are **not** copied to the forked view. However, depending
+automation, such as triggers, dmons, or cron jobs, are **not** copied to the forked layer. However, depending
```

Comment:
View was accurate here, those are view-specific things that aren't in a layer. Layers don't really get forked.

Reply:
DANGIT